v2/internal/pkg/operator/catalog_handler.go

```diff
@@ -207,13 +208,20 @@ func (o catalogHandler) filterRelatedImagesFromCatalog(operatorCatalog OperatorC
 func (o catalogHandler) getRelatedImagesFromCatalog(dc *declcfg.DeclarativeConfig, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) (map[string][]v2alpha1.RelatedImage, error) {
 	setInternalLog(o.Log)
 
+	var errs []error
+
 	relatedImages := make(map[string][]v2alpha1.RelatedImage)
 
 	for _, bundle := range dc.Bundles {
-		ris := handleRelatedImages(bundle, bundle.Package, copyImageSchemaMap)
+		ris, err := handleRelatedImages(bundle, bundle.Package, copyImageSchemaMap)
+		if err != nil {
+			internalLog.Warn("%s", err.Error())
```

Comment:
What about
```suggestion
			internalLog.Warn("SKIPPING bundle %s of operator %s: %s", bundle.Name, bundle.Package, err.Error())
```
then change the error returned by `handleRelatedImages` to not mention the operator/bundle?